components/edit-collective/sections/CollectiveGoals.js

```diff
@@ -48,7 +46,7 @@ class CollectiveGoals extends React.Component {
     this.defaultType = 'yearlyBudget';
     this.messages = defineMessages({
       add: { id: 'goal.add', defaultMessage: 'Add goal' },
-      remove: { id: 'goal.remove', defaultMessage: 'Remove goal' },
+      remove: { id: 'goal.remove', defaultMessage: 'Remove' },
```

Comment:
Let's use the generic one

```suggestion
      remove: { id: 'Remove', defaultMessage: 'Remove' },
```

Reply:
Done :+1: